docs/kubernetes/README.md

```diff
@@ -123,6 +123,21 @@ parameters:
 
 Your nodes might need to have `cryptsetup` installed to mount the volumes with LUKS.
 
+### Formatting Options
+
+You can specify extra formatting options which are passed directly to `mkfs.FSTYPE` via the `fsFormatOptions` parameter in the storage class.
+
+#### Example
+```yaml
+parameters:
+  csi.storage.k8s.io/fstype: xfs
+  fsFormatOptions: "-i nrext64=1"
+```
+
+### XFS Filesystem
+
+When using XFS as the filesystem type, we apply a default configuration to mkfs to ensure maximum compatibility with older Linux kernel versions.
```

Comment:
These docs should be a part of the general docs, they are not specific to Kubernetes. Happy to keep them here until we redo the docs though.